### DISCUSSION TITLE
imported the 22.1.0 code of dbeaver, but these errors occur.

### DISCUSSION THREAD

**Discussion by yangmeng66** [dbeaver/dbeaver#16916] [DBeaver Development]
Hello,
I imported the 22.1.0 code of dbeaver, but these errors occur. Please help me,

![image](https://user-images.githubusercontent.com/51857567/176402475-2c260d47-d439-417f-80a3-434cc61add04.png)

**Comment by xikeda-hunzi** [comment]
i faced the similar problem
<img width="392" alt="26eea06615516626c632acffadaf773" src="https://user-images.githubusercontent.com/91422515/176844274-b153149b-e462-417c-bfd3-d6392a91a2c9.png">

**Comment by miltonchirinos44** [comment]
@yangmeng66 
you must have java version 11 and maven 3.6.3 installed
